packages/web/pages/apps/index.tsx

```diff
@@ -0,0 +1,163 @@
+    const searchResults = fuse.search(value);
+    const appDataResults: appDataType[] = searchResults.map(
+      (result) => result.item as appDataType
+    );
+
+    setSearchValue(value);
+    setFuzzySearchResults(appDataResults);
+  };
+
+  const iterableData = searchValue ? fuzzySearchResults : nonFeaturedApps;
+
+  return (
+    <main className="m-auto max-w-container bg-osmoverse-900 py-3 md:px-3">
+      <div className="flex justify-between pl-6">
+        <div style={{ flexBasis: "55%" }}>
+          <h4 className="pb-2 text-h4 font-h4 text-wosmongton-100">
+            {t("store.headerTitle")}
+          </h4>
+          <span className="body2 text-osmoverse-200">
+            {t("store.headerSubtitle")}
+          </span>
+        </div>
+        <SearchBox
+          placeholder={t("store.searchPlaceholder")}
+          onInput={handleSearchInput}
+          className="self-end"
+          size="long"
+        />
+      </div>
+      <HeroCard
+        title={featuredApp.title}
+        subtitle={featuredApp.subtitle}
+        imageUrl={featuredApp.hero_image_URL}
+        githubUrl={featuredApp.github_URL}
+        twitterUrl={featuredApp.twitter_URL}
+        externalUrl={featuredApp.external_URL}
+        mediumUrl={featuredApp.medium_URL}
+      />
+      <div className="body2 mb-2 pt-7 pl-6 font-bold text-osmoverse-200">
+        All apps
+      </div>
+      <div className="container mx-auto py-3">
+        <div className="grid grid-cols-3 gap-4">
+          {iterableData?.map((app, index) => {
+            return (
+              <AppDisplayCard
+                key={index}
+                title={app?.title}
+                subtitle={app?.subtitle}
+                imageUrl={app?.thumbnail_image_URL}
+                twitterUrl={app?.twitter_URL}
+                githubUrl={app?.github_URL}
+                externalUrl={app?.external_URL}
+                index={index}
+              />
+            );
+          })}
+        </div>
+      </div>
+      <div className="container mx-auto flex py-6 pl-6">
+        <div className="flex flex-col pr-2" style={{ flexBasis: "35%" }}>
+          <h6 className="font-semibold">{t("store.getFeatured")}</h6>
+          <p className="pt-4 text-xs text-osmoverse-200">
+            {t("store.featuredDescription")}
+          </p>
+        </div>
+        <div className="flex w-48 items-center justify-center pl-6">
+          <Button mode="secondary" size="sm" onClick={handleApplyClick}>
+            {t("store.applyButton")} &rarr;
+          </Button>
+        </div>
+      </div>
+      <div className="flex w-full items-center overflow-x-auto rounded-lg bg-osmoverse-1000 px-8 py-6 text-osmoverse-400 2xl:gap-4 xl:gap-3 1.5lg:px-4 md:flex-col md:items-start md:gap-3 md:px-5 md:py-5">
+        <span className="text-xs">{t("store.storeDisclaimer")}</span>
+      </div>
+    </main>
+  );
+};
+
+export default AppStore;
+
+export async function getStaticProps() {
```

Comment:
FYI this is exactly what Next.js SSR rendering via static props was designed for. We get SEO benefits through SSR-generated HTML (with proper a tags, spans, etc.) while still being able to use React to allow the user to filter the page in the client, in this example.